ds/ds.go

```diff
@@ -125,12 +126,38 @@ func Parse(uri string) (dsQuery, error) {
 	}
 
 	// following dns-sd URI conventions from CUPS
-	if u.Host != "" {
-		result.Domain = u.Host
-	}
-	if u.Path != "" {
-		// remove prefix slash
-		result.Type = u.Path[1:]
+	//   (e.g. dnssd://instance._type._proto.domain/?query)
```

Comment:
Function comment should be of form
// Parse parses DNS-SD URI etc. etc. 

Hey, it ain't me, it's the various linters :-)

if you can make that tweak we are good to go.

Reply:
Done